exonum-java-binding/core/rust/exonum-java/start_cryptocurrency_node.sh

```diff
@@ -49,29 +46,29 @@ header "PREPARE PATHS"
 
 EJB_LOG_CONFIG_PATH="${EJB_APP_DIR}/log4j-fallback.xml"
 
-export LD_LIBRARY_PATH="$JVM_LIB_PATH"
+export LD_LIBRARY_PATH="$JAVA_LIB_DIR"
```

Comment:
I wonder if it is OK to use `JAVA_LIB_DIR` variable from the tests_profile :thinking: I considered setting RUSTFLAGS to be the only effect of tests_profile, not these intermediate variables. Shall we make it required to set this variable as well?

Reply:
These scripts are effectively protected from unset variables by `set -eu -o pipefail`. I think we can safely rely on the current tests_profile implementation, as these scripts are intended to be used by us (and any problem will be spotted just in time).